chore: remove unnecessary suffix from @vaadin/flow-frontend imports (#131)
vaadin/docs: articles/fusion/application/starting.asciidoc

----

For applications where the UI is built in Java and does not include any client-side views the client-side bootstrapping process is transparent.

> A default `index.html` / `index.ts` pair is generated during the build, and loads the server-side `UI` when the app starts.

Reviewer: ```suggestion
A default `index.html` / `index.ts` pair is generated during the build, and loads the server-side `UI` when the application starts.
```

> If there is a need to add a client-side view to the application later, you can do it incrementally, with minimal modifications.

Reviewer: ```suggestion
Client-side views can be added incrementally to the application later, with minimal modifications.
```

----

> With client-side routing you can optimize your application to load the minimal content to be displayed to users at the first load. 
> This might reduce the first-loading time compared to the Vaadin 10–14 bootstrapping method which loads a full Vaadin application on the first request. 

Reviewer: ```suggestion
Client-side routing can improve the first load/startup of the application, by minimizing the content delivered to the browser on initial load.
```
I don’t think we need to talk negatively about Vaadin 10–14 bootstrapping. Let’s just point out the benefit of using client-side routes.

Author: good point

> As a result, the first request loads promptly and that improves the first interaction experience.

Reviewer: ```suggestion
```
Redundant

== Bootstrap Page Template [[bootstrap-page-template]]

> When using client-side bootstrapping, Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. 

Reviewer: ```suggestion
When using client-side bootstrapping, the Vaadin servlet uses the `frontend/index.html` file as a template to generate the bootstrap page response. 
```

> It processes the template and injects the following information:

Reviewer: ```suggestion
The servlet processes the template and injects the following information:
```

  - `<base href='./relative/to/root'>`: Vaadin calculates the relative path from the current request path to the root path of the application. 

>   That is required for relative links in the app view templates to work correctly.

Reviewer: ```suggestion
  That is required for relative links in view templates to work correctly.
```

  - Bundled script: Vaadin automatically adds the bundled and optimized script generated from the `frontend/index.ts` file. 

>   It uses a pre-configured link:https://webpack.js.org/[Webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. 

Reviewer: ```suggestion
  It uses a pre-configured link:https://webpack.js.org/[webpack^] instance that's included together with `vaadin-maven-plugin` as a module bundler. 
```

>   Therefore, in the `frontend/index.html` template there is no need to include the `index.ts` (or `index.js`) script manually.

Reviewer: ```suggestion
  Therefore, the `frontend/index.html` template does not need to include the `index.ts` (or `index.js`) script explicitly.
```

NOTE: The frontend directory can be customized by providing the property `vaadin.frontend.frontend.folder` when running the Maven goals `prepare-frontend`  or `build-frontend` from `vaadin-maven-plugin`.

=== Default Bootstrap Template and Entry Point [[default-bootstrap-template-and-entry-point]]

If the `index.html` or `index.ts` files in the frontend folder are missing, `vaadin-maven-plugin` generates a default corresponding file in the `target` folder. 

> The generated files starts the application so that it uses only server-side routing. 

Reviewer: ```suggestion
In that case the application uses server-side routing only.
```

router.setRoutes(routes);
----

> NOTE: The auto-generated bootstrap template works best for server-side applicaitons. 

Reviewer: ```suggestion
.The defaults are best for Flow applications
[NOTE]
The autogenerated bootstrap template works best for server-side applications. 
```

> To add client-side views into the app, please follow the  <<quick-start-guide#step-3, Quick Start Guide>>.

Reviewer: ```suggestion
To add client-side views into the application, follow the <<quick-start-guide#step-3, Quick Start Guide>>.
```
